src/main.c

```diff
@@ -1172,8 +1172,14 @@ int main(int arg_c, char **arg_v)
     dcc[term_z].sock = STDOUT;
     dcc[term_z].timeval = now;
     dcc[term_z].u.chat->con_flags = conmask | EGG_BG_CONMASK;
-    dcc[term_z].u.chat->strip_flags = STRIP_ALL;
     dcc[term_z].status = STAT_ECHO;
+    if (isatty(dcc[term_z].sock)) {
+      putlog(LOG_DEBUG, "*", "stdout is a tty");
+      dcc[term_z].status |= STAT_TELNET;
+    } else {
+      putlog(LOG_DEBUG, "*", "stdout is no tty");
```

Comment:
should be debug0 probably, because no one needs to see this in production, right?

Reply:
fixed